apertium-eng.eng.dix

```diff
@@ -15938,7 +15938,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <e lm="liveliness">      <i>liveliness</i><par n="access__n"/></e>
 <e lm="liveness">        <i>liveness</i><par n="access__n"/></e>
 <e lm="liver">           <i>liver</i><par n="house__n"/></e>
-<e lm="livermorium"> <i>livermorium</i><par n="house__n"/></e>
+<e lm="livermorium">     <i>livermorium</i><par n="house__n"/></e>
```

Comment:
Ok so now the spacing issue is fixed. Could you do this for every other word you added please?

Reply:
ok